components/map/Map.tsx

```diff
@@ -0,0 +1,54 @@
+import { MapContainer, TileLayer, Marker, Popup, useMap } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import "leaflet-defaulticon-compatibility";
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
+import { Company } from "@prisma/client";
+import styles from "./Map.module.css";
+import Link from "next/link";
+
+type MapProps = {
+  companies: Company[];
+};
+
+const Map: React.FC<MapProps> = ({ companies, ...props }) => {
+  // const position = [companies[0].latitude, companies[0].longitude]
+  // console.log("companies", companies[0].latitude, companies[0].longitude)
+  return (
+    <MapContainer
+      style={{
+        height: "100vh",
+      }}
+      center={[52.136, 5.2913]}
```

Comment:
Idea for a future improvement: 

- Center the map on the average coordinates of the results, this way if you choose a city like "amsterdam" that's where the map will focus